http/server.go

```diff
@@ -40,7 +41,17 @@ func New(ctx context.Context, client drand.PublicClient, logger log.Logger) (htt
 	if logger == nil {
 		logger = log.DefaultLogger
 	}
-	handler := handler{reqTimeout, client, parsedPkt, logger}
+	handler := handler{
```

Comment:
Maybe the struct can be renamed to `httpHandler`?

Reply:
go lint will complain that `httpHandler` stutters, since it'll be referenced outside this package as `http.httpHandler`

Reply:
Ah, yeah ok, it just reads weirdly to me.